### PR TITLE
Add "EXTERNAL" to possible recurring.contract values

### PR DESCRIPTION
**Pull request by crrood**
To support third party network tokens

**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

**Comment by crrood** [comment]
Hey guys,

Looks like Travis is failing due to a CheckStyle config.. could be I don't know the proper format for adding an enum:
```
Failed during checkstyle configuration: Exception was thrown while processing /home/travis/build/Adyen/adyen-java-api-library/src/main/java/com/adyen/model/recurring/Recurring.java: MismatchedTokenException occurred during the analysis of file /home/travis/build/Adyen/adyen-java-api-library/src/main/java/com/adyen/model/recurring/Recurring.java. expecting EOF, found 'ContractEnum'
```

File: src/main/java/com/adyen/model/recurring/Recurring.java [hunk 1]
```diff
@@ -58,6 +58,9 @@
 
         @SerializedName("RECURRING,PAYOUT")
         RECURRING_PAYOUT("RECURRING,PAYOUT");
```
Comment (Aleffio) [inline src/main/java/com/adyen/model/recurring/Recurring.java:60]:
```suggestion
        RECURRING_PAYOUT("RECURRING,PAYOUT"),
```

File: src/main/java/com/adyen/model/recurring/Recurring.java [hunk 2]
```diff
@@ -58,6 +58,9 @@
...
+        
+        @SerializedName("EXTERNAL")
+        RECURRING_PAYOUT("EXTERNAL");
```
Comment (Aleffio) [inline src/main/java/com/adyen/model/recurring/Recurring.java:63]:
```suggestion
        EXTERNAL("EXTERNAL");
```

**Comment by crrood** [comment]
Hey @Aleffio,

Thanks for fixing the commit, although it looks like it's still failing due to some checkstyle rules.  I'll email you separately with more info on the request and EXTERNAL value.

**Comment by martinsrenato** [comment]
Hey @crrood ,
We just merged #337, which had the same fix, but no checkstyle breaking changes.
I'll close this one for now.

**Pull request closed without merging**